src/inflators/loop-animation.ts

```diff
@@ -0,0 +1,60 @@
+import { addComponent } from "bitecs";
+import { LoopAnimationInitialize, LoopAnimationInitializeData } from "../bit-components";
+import { HubsWorld } from "../app";
+
+type ElementParams = {
+  activeClipIndex?: number;
+  // TODO: Do we need to keep supporting the following two params?
+  //       Perhaps we should detemine which one should be the
+  //       the canonical presentation and then handle converting
+  //       the others to that.
+  // DEPRECATED: Use activeClipIndex instead since animation
+  //             names are not unique
+  clip?: string;
+  // Support for Spoke->Hubs activeClipIndices struct
+  activeClipIndices?: number[];
+  paused?: boolean;
+  startOffset?: number;
+  timeScale?: number;
+};
+
+export type LoopAnimationParams = ElementParams[];
```

Comment:
Was quite confused why this is an array, but I see in model inflator that you are attempting to consolidate all the loop-animation components in a model into a single one. I think that makes some sense. It's the component we would like to have. Though if we are going to do that anyway, I think the component data should be consolidated earlier too. Maybe it should just be an array of clips or actions (since that's what you end up doing in the system anyway). That way we can move the messy handling of the legacy properties all up front and not have the rest of the code need to think about it.